doc/api/buffer.md

```diff
@@ -327,7 +326,7 @@ Allocates a new `Buffer` using an `array` of octets.
 const buf = new Buffer([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 ```
 
-### new Buffer(arrayBuffer[, byteOffset [, length]])
+### new Buffer(arrayBuffer[, byteOffset[, length]])
```

Comment:
Hmmm...looks like my above "this is different than other argument formatting" was not exactly universally correct. This line as it exists seems to use both space and no space....